src/SingleCellAnnotator.imjoy.html

```diff
@@ -416,6 +421,7 @@
       const response = await fetch(url_post, {
           method: 'POST',
           mode: 'cors',
+          credentials: 'include',
```

Comment:
Note, starting now, we don't use credentials, please make sure you integrate the changes in #28 : https://github.com/CellProfiling/cellpro-imjoy-plugins/pull/28/files#diff-41fa0174fd16cdd8ccbab0a82b5d597dR212

Reply:
it cannot work with post for annotation, if dont include that. As I ve tested with casper

Reply:
got it